### PR TITLE
add release notes for v1.23.0

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

**Review by Rugvip — APPROVED** [review]
🎉

File: docs/releases/v1.23.0.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+The `@backstage/eslint-rules` package now provides the `@backstage/no-top-level-material-ui-4-imports` rule that forbids imports such as `import { Box } from '@material-ui/core'`. This rule is currently disabled by default and needs to be manually enabled in `.eslintrc.js`:
+
+```json
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname, {
+  rules: {
+    '@backstage/no-top-level-material-ui-4-imports': 'error',
+  },
+});
+
```
Comment (awanlin) [inline docs/releases/v1.23.0.md:43]: Missing ending backticks?

File: docs/releases/v1.23.0.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+The `backstage-cli repo fix` command will now update the `repository` fields in all workspace packages, as long as the root `package.json` has the `repository` field set.
```
Comment (awanlin) [inline docs/releases/v1.23.0.md:44]: Does this need a title?

File: docs/releases/v1.23.0.md (hunk 1 above)
Comment (freben) [reply docs/releases/v1.23.0.md:44]: fixed

File: docs/releases/v1.23.0.md (hunk 2 above)
Comment (freben) [reply docs/releases/v1.23.0.md:47]: ah quite right, the diff was misleading above

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**